en/flight_controller/omnibus_f4_sd.md

```diff
@@ -171,12 +171,19 @@ Instructions for this are provided in the [TBS Crossfire Manual](https://www.tea
 
 The schematics are provided by [Airbot](https://myairbot.com/): [OmnibusF4-Pro-Sch.pdf](http://bit.ly/obf4pro).
 
-## Build & Upload Instructions
+## Build & Upload Instructions{#upload}
 
 The board comes pre-installed with [Betaflight](https://github.com/betaflight/betaflight/wiki). 
-Before PX4 firmware can be installed, the PX4 bootloader needs to be built and uploaded.
+Before the PX4 firmware can be installed, the PX4 bootloader needs to be flashed.
```

Comment:
Does this process *just* flash the bootloader, or does the .hex file you refer to below also include firmware? This should be clarified, because on the whoop docs the assumption is that it does. If it doesn't then the first step on QGC must be to install PX4.

Reply:
Only the bootloader.

Reply:
@bkueng So now I am very confused. If Betaflight configurator can only install the bootloader, and as above https://github.com/PX4/px4_user_guide/pull/414/files#r247034388 you can't then use QGC to install the firmware, then how do you get firmware onto the vehicle?

Or is it the case that you use configurator to put the bootloader on, then rather than using QGC to upload the firwmare (ie via custom firmware upload) you must do a make with `upload` option to get firmware on board?

Reply:
> Or is it the case that you use configurator to put the bootloader on, then rather than using QGC to upload the firwmare (ie via custom firmware upload) you must do a make with upload option to get firmware on board?

Exactly. That's how everyone has done it so far (we devels never use QGC to upload). Now you can: https://github.com/mavlink/qgroundcontrol/pull/7142.